src/main/java/mate/academy/hibernate/relations/dao/impl/ActorDaoImpl.java

```diff
@@ -12,11 +14,32 @@ public ActorDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Actor add(Actor actor) {
-        return null;
+        Session session = null;
+
+        try {
+            session = factory.openSession();
+            session.beginTransaction();
+
+            session.persist(actor);
+
+            session.getTransaction().commit();
+        } catch (Exception e) {
+            throw new DataProcessingException("Cannot create actor", e);
```

Comment:
you should make a transaction rollback 
```suggestion
            if (transaction != null) {
                transaction.rollback();
            }
            throw new DataProcessingException("Cannot create actor", e);
```